README.md

```diff
@@ -9,17 +9,30 @@ This is the Glitch equivalent of running `create-probot-app` to generate a new p
 
 To get your own Glitch-hosted Probot up-and-running:
 
-1. Head on back to our [Devlopment Docs](https://probot.github.io/docs/development/#configuring-a-github-app) to configure your GitHub App.
 
-2. Click the **New File** button (at left on Glitch) and type `.data/private-key.pem`. Then click **Add File**. Open the private key you downloaded during your Github App creation, and copy/paste the contents into your new file.
+1. [Configure a new app on Github](https://github.com/settings/apps/new).
+    - Hit the "Show" button on the top left of this page to find the URL. It will look something like `https://random-word.glitch.me/probot`, except the domain will be specific to your app.
+    - For the Homepage URL, use your app URL that you just found (updating the domain to match yours): `https://random-word.glitch.me/probot`
+    - For the Webhook URL, use this URL (again, updating the domain to match yours): `https://random-word.glitch.me/`. Notice that we left off the `/probot`.
+    - For the Webhook Secret, open a terminal and run `openssl rand -base64 32`. Copy/paste the outputted value to the Webhook Secret box. Keep this handy
+    until Step 4.
+    - Choose the permissions you want to give your bot based on what you want to build (ex. issues bot, PR bot, hybrid).
+    - Save your changes.
+    - On the configuration page that comes up after saving, download your private key. It will be saved into a file named `my-app-name.2018-06-20.private-key.pem`, with your app name, and today's date.
 
-3. Edit the `.env` file (at left) with your app credentials. 
+2. Click the **Install** tab, and install your app into one of your repositories.
+
+3. Click the **New File** button (at left) and type `.data/private-key.pem`. Then click **Add File**. Open a terminal, and from your download folder run `cat my-app-name.2018-06-20.private-key.pem | pbcopy` (except using your app's name and today's date). In the new file in Glitch, paste the contents of the clipboard.
```

Comment:
Do you think this is easier than just opening up the private key in any text editor?